tests/lib/store.sh

```diff
@@ -19,7 +19,9 @@ setup_fake_store(){
     # debugging
     systemctl status fakestore || true
     echo "Given a controlled store service is up"
-    systemd-run --unit fakestore $(which fakestore) -start -dir $top_dir -addr localhost:11028 $@
+    https_proxy=${https_proxy:-}
+    http_proxy=${http_proxy:-}
```

Comment:
Yes, we need it because we use "set -u" in our tests, bash will complain about an undefined variable if it is not set.